generate_html.rb

```diff
@@ -41,6 +41,8 @@
     configuration[:display_cookies] = false
   end
 
+  configuration[:participants] = YAML.load_file('participants.yml').map { |participant| Hash[participant.map { |k,v| [k.to_sym,v] }] } if File.exists? 'particiants.yml'
```

Comment:
is this change to deal with the yml file not being present?

Reply:
yep